demos/shapes-custom-question-authoring-demo/src/question/index.js

```diff
@@ -0,0 +1,339 @@
+            el.querySelector(".lrn_response_input").classList.remove("lrn_correct");
+            el.querySelector(".lrn_response_input").classList.remove("lrn_incorrect");
+            
+            // // if we chose to also display the correct answer, we will now remove the display of the correct answer as well by 
+            // // leveraging the suggestedAnswersList.reset() method.
+            this.suggestedAnswersList.reset()
+        };
+    }
+
+    handleEvents() {
+        const { events, init, el } = this;
+        const { question } = init;
+        const facade = init.getFacade()
+
+        // TODO: Requires implementation - Make sure you trigger 'changed' event after the user changes their responses to your custom quesiton:
+
+        /**
+         * 
+         * When you consider that the response to your question has been changed, 
+         * (such when a user clicks the choice of a multiple choice question or enters text into a free response question),
+         * then you must trigger the changed event to inform Questions API that the question has been attempted.
+         * 
+         * To do this, you must call events.trigger('changed', responses);  
+         * This call should take place inside of a DOM event listener in which the user has changed their response to your question
+         * (such as a 'click' event listener for a multiple choice question, or an input 'change' event listener for a free response text question)
+         * 
+         * @param responses The 'responses' variable can be any data type you would like it to be depending on the format of your custom question's responses. 
+         * For instance, it might be a string for the value of a simple text input
+         * Or for a quesiton asking a student to plot coorinates on a graph, it might be an object containing the x and y coordinates, e.g.
+         *     {
+         *       x: 123,
+         *       y: 408
+         *     }
+         */
+
+        /** EXAMPLE implementation
+         * 
+         * document.getElementById('my-radio-button').addEventListener('click', (event) => {
+         *   
+         *  // STEP 1: when the user clicks on your custom question's input, save its current value as the value of responses
+         *  let responses = event.target.value
+         *   
+         *  // STEP 2: 
+         *  // now notfy Questions API that the user has changed their response
+         *   
+         *   events.trigger('changed', responses);
+         * 
+         *  // STEP 3 - OPTIONAL
+         *  // If you have chosen to implement facade.showValidationUI() above to display visiual feedback for a correct / incorrect answer
+         *     when the user presses "Check Answer" for the current response:
+         *     then make sure you have also implemented facade.resetValidationUI(), and that you call it here:
+         * 
+         * // facade.resetValidationUI()
```

Comment:
Fix comments for readability
```suggestion
         *  STEP 1: when the user clicks on your custom question's input,
         * save its current value as the value of responses
         *    let responses = event.target.value
         *   
         *  STEP 2: 
         *  now notify Questions API that the user has changed their response
         *   
         *   events.trigger('changed', responses);
         * 
         *  STEP 3 - OPTIONAL
         *  If you have chosen to implement facade.showValidationUI() above to display visual
         *  feedback for a correct / incorrect answer
         *  when the user presses "Check Answer" for the current response:
         *  then make sure you have also implemented facade.resetValidationUI(), and that you call it here:
         * 
         *   facade.resetValidationUI()
```